docs/tutorial/flows.md

```diff
@@ -9,24 +9,29 @@ tags:
     - parameters
     - retries
 ---
-## What is a Flow?
+!!! note "Start a Prefect API and UI"
+    This tutorial is best paired with a Prefect UI so that you can see the information that Prefect is capturing.  If using a self-hosted setup, simply run `prefect server start` to run both the webserver and UI; if using Prefect Cloud, navigate to your workspace at [`https://app.prefect.cloud/`](https://app.prefect.cloud/).
```

Comment:
```suggestion
    This tutorial is best paired with a Prefect UI so that you can see the information that Prefect is capturing.  
    If using Prefect Cloud, navigate to your workspace at [`https://app.prefect.cloud/`](https://app.prefect.cloud/).
    If using a self-hosted setup, run `prefect server start` to run both the webserver and UI. 
```